PendingReleaseNotes

```diff
@@ -424,3 +424,6 @@
   (``auth_client_required=cephx`` and ``ms_mon_client_mode=secure``).
   If you have cephx authentication disabled on your cluster, you may
   need to adjust these settings for RGW.
+
+* LevelDB support has been removed. ``WITH_LEVELDB'' is no longer a supported
+  build option.
```

Comment:
could you please add this under the `>=17.0.0` section https://github.com/ceph/ceph/blob/67d75cf37de5bf92c869c744d85a5a41fb8e7bed/PendingReleaseNotes#L1-L82

Reply:
Done, sorry. I didn't notice the other headings and thought the whole file was >=17

Reply:
no worries, thanks!